doc/nrf/libraries/bootloader/bl_monotonic_counters.rst

```diff
@@ -0,0 +1,47 @@
+.. _doc_bl_monotonic_counters:
+
+Monotonic counters
+##################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The monotonic counters library implements the logic for readind and writing uint16_t counters which can only increase.
+
+The library is used by :ref:`bootloader` to keep track of the version of the application.
+The monotonic counters are stored as part of the bootloader storage :ref:`doc_bl_storage`.
```

Comment:
This is misleading.

In the MCUBoot->app use-case where BL_STORAGE is disabled the counters are not stored as part of the booltoader storage.

Perhaps this section should go into the https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/library_template_README.html#samples-using-the-library section and be explicit about this only being the case for when NSIB uses the library.

Reply:
This sentence is referring the bootloader sample (NSIB), not MCUBOOT. I will try to rephrase it.